src/shared/feature-flags/flags-status-table.tsx

```diff
@@ -55,15 +59,29 @@ export const FlagsStatusTable: React.FC<Props> = ({ data }) => {
     <ContentPadding>
       <CommonMargin />
       <h1>Feature Flags</h1>
-      <NewFlagController />
+      <NewFlagController newFlaglabel="+ Flag" />
       <div>
         Use this page to see all feature flags in this project. Select a flag to
         manage the environment-specific targeting and rollout rules.
       </div>
-      <Table<FlagsStatus_flagsStatus_flags>
-        columns={columns}
-        dataSource={data}
-      />
+      <ConfigProvider
```

Comment:
why use ConfigProvider here?

Reply:
https://stackoverflow.com/questions/42186723/antd-ui-library-overriding-table-behavior-on-empty-data

here with ConfigProvider we can overriding emptyData in Table .